tests/realm.cpp

```diff
@@ -152,6 +152,18 @@ TEST_CASE("SharedRealm: get_shared_realm()") {
         }
     }
 
+    SECTION("should be able to set a FIFO fallback path") {
+        TestFile config;
+        config.fifo_files_fallback_path = "fallback_path";
+        config.schema_version = 1;
+        config.schema = Schema{
+            {"object", {
+                {"value", PropertyType::Int}
+            }},
+        };
+        REQUIRE_NOTHROW(Realm::get_shared_realm(config));
```

Comment:
This doesn't actually test anything? Should at least verify that the files are in the expected path.

Reply:
I know, this was mostly a sanity check. I expected that core would only write the files to the backup location in case the primary location failed. If I misunderstood that, then yes, we should check the files